XenModel/Actions/Updates/DownloadUpdatesXmlAction.cs

```diff
@@ -83,10 +83,10 @@ public DownloadUpdatesXmlAction(bool checkForXenCenter, bool checkForServerVersi
             _checkForXenCenter = checkForXenCenter;
             _checkForServerVersion = checkForServerVersion;
             _checkForPatches = checkForPatches;
-            _checkForUpdatesUrl = string.IsNullOrEmpty(checkForUpdatesUrl) ? InvisibleMessages.XENSERVER_UPDATE_URL : checkForUpdatesUrl;
+            _checkForUpdatesUrl = checkForUpdatesUrl;

```

Comment:
Following a quick discussion on this, we agreed I will put an Assert in for null only.